plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c

```diff
@@ -8,17 +8,17 @@
 
 #include <string.h>
 
-#include "fu-unifying-hidpp.h"
-#include "fu-unifying-hidpp-msg.h"
+#include "fu-logitech-hid-hidpp.h"
+#include "fu-logitech-hid-hidpp-msg.h"
 
-FuUnifyingHidppMsg *
-fu_unifying_hidpp_msg_new (void)
+FuLogitechHidHidppMsg *
+fu_logitech_hid_hidpp_msg_new (void)
```

Comment:
hid_hidpp looks kinda odd to me -- maybe `FuLogitechHidppMsg` instead?

Reply:
Aaaaactually, I wonder if the plugin name should be logitech-hidpp?

Reply:
It will also have the bootloader protocols, but I guess that's pretty small.

Reply:
I just pushed a fixup to rename to logitech-hidpp instead then.